guides/plugins/plugins/administration/add-rule-assignment-configuration.md

```diff
@@ -0,0 +1,256 @@
+# Add rule assignment configuration
+
+## Overview
+
```

Comment:
I think a screenshot would really help people understand what this guide is about. I can't really picture what we're going to do here as I start reading this 🤔